vita3k/gui/src/app_context_menu.cpp

```diff
@@ -103,36 +103,35 @@ static bool get_update_history(GuiState &gui, HostState &host) {
     return !update_history_infos[host.io.title_id].first.empty() && !update_history_infos[host.io.title_id].second.empty();
 }
 
-static std::map<std::string, std::string> game_info;
+static std::map<std::string, std::string> app_info;
 
-static bool get_game_info(GuiState &gui, HostState &host) {
-    game_info.clear();
-    const auto game_path{ fs::path(host.pref_path) / "ux0/app" / host.io.title_id };
+static bool get_app_info(GuiState &gui, HostState &host) {
+    app_info.clear();
+    const auto app_path{ fs::path(host.pref_path) / "ux0/app" / host.io.title_id };
 
-    if (fs::exists(game_path) && !fs::is_empty(game_path)) {
-        game_info["trophy"] = fs::exists(game_path / "sce_sys/trophy") ? "Eligible" : "Ineligible";
+    if (fs::exists(app_path) && !fs::is_empty(app_path)) {
+        app_info["trophy"] = fs::exists(app_path / "sce_sys/trophy") ? "Eligible" : "Ineligible";
 
-        time_t updated = fs::last_write_time(game_path);
-        game_info["updated"] = std::asctime(std::localtime(&updated));
+        const auto last_writed = fs::last_write_time(app_path);
+        const auto updated = std::localtime(&last_writed);
+        app_info["updated"] = fmt::format("{}/{}/{} {:0>2d}:{:0>2d}", updated->tm_mday, updated->tm_mon + 1, updated->tm_year + 1900, updated->tm_hour, updated->tm_min);
 
-        size_t game_size = 0;
-        fs::recursive_directory_iterator end;
-        for (fs::recursive_directory_iterator g(game_path); g != end; ++g) {
-            const fs::path file = *g;
-            if (fs::is_regular_file(file))
-                game_size += fs::file_size(file);
+        size_t app_size = 0;
+        for (const auto &app : fs::recursive_directory_iterator(app_path)) {
+            if (fs::is_regular_file(app.path()))
+                app_size += fs::file_size(app.path());
         }
-        game_info["size"] = std::to_string(game_size / 1048576);
+        app_info["size"] = std::to_string(app_size / 1048576);
```

Comment:
Hey zang while you're at it.
```suggestion
        app_info["size"] = std::to_string(app_size / MB(1));
```

Reply:
AH lol